eclass/llvm.eclass

```diff
@@ -64,6 +64,8 @@ esac
 if [[ ! ${_LLVM_ECLASS} ]]; then
 _LLVM_ECLASS=1
 
+IUSE="bootstrap-prefix"
```

Comment:
It sounds like a serious overkill to add a bootstrap flag to all packages using LLVM in any way.

Reply:
Updated, PTAL